CarpetLib/src/prolongate_3d_dgfe_rf2.cc

```diff
@@ -239,8 +239,8 @@ void prolongate_3d_dgfe_rf2(
     int const srcstr[3] = {srcdi, srcdj, srcdk};
     int const dststr[3] = {dstdi, dstdj, dstdk};
 
-// Loop over fine region
-#pragma omp parallel for collapse(3)
+    // Loop over fine region
+    // #pragma omp parallel for collapse(3)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.